share/picongpu/examples/Bunch/etc/picongpu/32.cfg

```diff
@@ -66,7 +66,7 @@ TBG_plugins="!TBG_eBin                     \
 
 TBG_deviceDist="!TBG_devices_x !TBG_devices_y !TBG_devices_z"
 
-TBG_programParams="-d !TBG_deviceDist \
+TBG_programParams="--openPMD.period 1000 --openPMD.file simData%T.bp -d !TBG_deviceDist \
```

Comment:
prefer `simData_%06T.bp`, put this into `plugins` and pls keep `-d !TBG_deviceDist` on a separate line